#!/bin/sh -e
## 35_updatedb-location.dpatch by Thomas Schoepf <schoepf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change updatedb(1) to reflect the fact that the database now lives
## DP: in /var/cache. Bug#171877

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/ametzler/TIN/FINDUTILS/findutils-4.1.20/locate/updatedb.1 findutils-4.1.20/locate/updatedb.1
--- /home/ametzler/TIN/FINDUTILS/findutils-4.1.20/locate/updatedb.1	Sat Nov 28 18:29:39 1998
+++ findutils-4.1.20/locate/updatedb.1	Sun Jul 13 14:18:41 2003
@@ -67,7 +67,7 @@
 .B \-\-output=\fIdbfile\fP
 The database file to build.
 Default is system-dependent.  In Debian GNU/Linux, the default
-is /var/lib/locate/locatedb.
+is /var/cache/locate/locatedb.
 .TP
 .B \-\-localuser=\fIuser\fP
 The user to search non-network directories as, using \fBsu\fP(1).
